src/views/ops/Adhoc/QuickJob.vue

```diff
@@ -475,6 +484,17 @@ export default {
       if (this.parameters) {
         data.parameters = this.parameters
       }
+      this.showConfirmRunAssetsDialog = true
+      this.$axios.post('/api/v1/ops/inventory/classified-hosts/', {
+        assets: hosts,
+        nodes: nodes,
+        module: this.module,
+        args: this.command,
+        runas: this.runas,
+        runas_policy: this.runasPolicy
+      }).then(data => {
+        this.classifiedAssets = data
+      })
       createJob(data).then(res => {
         this.executionInfo.timeCost = 0
         this.executionInfo.status = { value: 'running', label: this.$t('Running') }
```

Comment:
Based on the provided code snippet for both `Page` component with its children components such as `QuickJobTerm`, `AdhocOpenDialog`, `AdhocSaveDialog`, `VariableHelpDialog`, and `NodeTree`. There do not appear to be specific issues or irregularities found within that codebase at the point of reviewing it currently. It seems well structured and functional. However, since you specifically requested feedback on differences between current versions versus a past version, without more context about when these changes were made I cannot analyze the differences accurately. Please provide additional details so I can offer useful information.